src/components/navbarItems/NavbarItems.tsx

```diff
@@ -1,24 +1,40 @@
-import {  MouseEventHandler } from 'react'
+import {  MouseEventHandler, useEffect, useState } from 'react'
 
 import { NavLink } from 'react-router-dom'
 import { AiOutlineHome as HomeIconForNavbar, AiOutlinePlusCircle as PlusIconForNavbar } from 'react-icons/ai'
 import { IoPersonOutline as AdminIconForNavbar } from 'react-icons/io5'
-
+import jwtDecode from 'jwt-decode'
 
 import './NavbarItems.css'
+import TokenService from '../../services/TokenService'
+import { isUserAdmin } from '../../utils/Roles'
+import { roleKey, Jwt } from '../../model/JWT'
 
-interface OpenModal {
+interface NavbarItemsProps {
   setModalIsOpen: MouseEventHandler<HTMLAnchorElement>
+  isLogged: boolean
 }
 
-const NavbarItems = ({ setModalIsOpen }: OpenModal) => {
+const NavbarItems = (props : NavbarItemsProps ) => {
+  const [ isAdmin, setIsAdmin ] = useState(false)
 
+  useEffect(() => {
+    const token = TokenService.getAccesToken()
+    if(token) {
+      setIsAdmin(isUserAdmin(jwtDecode<Jwt>(token)[roleKey]))
+    }
+  }, [])
   return (
+    <>
```

Comment:
Why do you need this wrapper if you have navbar_item_container already?

Reply:
Because i have conditional rendering whitch start with curly brackets and I think that react require wrapper.